README.md

```diff
@@ -201,7 +202,7 @@ import tensorflow_cloud as tfc
 tfc.run(entry_point='mnist_example.py')
 ```
 
-Please note that all the files in the same directory tree as `entry_point` will be packaged in the docker image created, along with the `entry_point` file.
+Please note that all the files in the same directory tree as `entry_point` will be packaged in the docker image created, along with the `entry_point` file. It's recommended to create a new directory to house each cloud project which includes necessary files and nothing else, to optimize image build times. 
```

Comment:
Please add/update this doc under workflow 2 and 3 as well.

Reply:
Looks like you have updated only for workflow 2, can you add for workflow 3 as well?